adafruit_bitmap_font/pcf.py

```diff
@@ -133,27 +139,27 @@ def __init__(self, f, bitmap_class):
         )
 
     @property
-    def ascent(self):
+    def ascent(self) -> int:
         """The number of pixels above the baseline of a typical ascender"""
         return self._ascent
 
     @property
-    def descent(self):
+    def descent(self) -> int:
         """The number of pixels below the baseline of a typical descender"""
         return self._descent
 
-    def get_bounding_box(self):
+    def get_bounding_box(self) -> tuple:
         """Return the maximum glyph size as a 4-tuple of: width, height, x_offset, y_offset"""
         return self._bounding_box
 
-    def _read(self, format_):
+    def _read(self, format_: str) -> tuple:
```

Comment:
Same thing with `Tuple` here, but it looks like we can define the contents since it varies, so no worries about defining the contents.